### ISSUE TITLE
nbTagInput is always autofocus and trigger ExpressionChangedAfterItHasBeenCheckedError false to true

### ISSUE THREAD

**Issue by wadinj** [akveo/nebular#2723]
### Issue type

**I'm submitting a ...**  (check one with "x")

* [x] bug report
* [ ] feature request

### Issue description

**Current behavior:**
When using the nbTagInput with autocomplete. The input is always autofocus. Even on the [nebular demo page](https://akveo.github.io/nebular/docs/components/tag/examples#nbtaginputdirective) this is the case for nbTagInput with autocomplete. This autofocus raises the error ExpressionChangedAfterItHasBeenCheckedError. focus false to true

**Expected behavior:**
The input should not be auto focused

**Steps to reproduce:**
Taking [this example](https://akveo.github.io/nebular/docs/components/tag/examples#nbtaginputdirective) from the nebular component page with autocomplete 

**npm, node, OS, Browser**
```
npm 7.7.6
node 15.14.0
MacOS Catalina
Chrome 90.0.4430.85
```

**Angular, Nebular**
```
Angular 10.1.1 & Nebular 7.0.0
```

**Comment by wadinj** [comment]
As a workaround I created a directive which delay a bit the display of the tag input. This is not a cleanest solution but a nice workaround if you're facing this issue
`import {Directive, ElementRef} from '@angular/core';

@Directive({
  selector: '[appDelay]'
})
export class DelayDirective {

  constructor(el: ElementRef) {
    el.nativeElement.style.display = 'none';
    setTimeout(() => el.nativeElement.style.display = 'inherit', 20);
  }

}
`

then attach this directive on the nb-form-field:
`<nb-form-field appDelay>`

**Comment by RodrigoAN97** [comment]
The solution @wadinj provided works in some cases, in others don't. This issue should't be closed.

**Comment by yggg** [comment]
@RodrigoAN97 The #2734 fix landed in the `8.0.0` should fix the issue. Did you check it in the 8.0.0 version? If the issue is still there after the fix, please, post a code example where it comes out and I'll reopen the issue.
